introduction.md

```diff
@@ -0,0 +1,101 @@
+# :tada: Welcome to the community! :tada:
+
+We are glad that you have decided to join us.
+There a few things that we have documented here to help you along from the beginning because we are grand believers in the Don't Repeat Yourself principle, and it just seems so professional!  
+
+Dont forget to Introduce Yourself in the #introductions channel. Please provide your interests, hobbies, organization, or whatever. You can divulge as much as you want.  This is a good way to summon others of like interests. :wink:
+
+Example:
+```
+Hello, everyone! My name is Ocat and I'm a full stack engineer @TacoBot My personal tech passions are working with APIs and giving out :taco:s. 
+I like to give back to the open source community when I can. PS. I love asyncio. 
+```
+
+## Channels
+If you are new to Slack then you should be aware of the concept of channels. Channels are a good way to keep conversations under the :umbrella: of a specific topic or context. Channels appear on the upper left side bar of the Slack UI.
+
+Channel topics can be specific to the python packages you :heart: like #django, #flask, #sqlalchemy, etc. We also have channels that are for learning like #help and #learningpython. 
```

Comment:
TODO: we need to try the slack [deeplinks](https://api.slack.com/docs/deep-linking) (I will file an issue to keep it separate)